pkgs/top-level/all-packages.nix

```diff
@@ -32876,6 +32880,8 @@ with pkgs;
 
   legendary-gl = python38Packages.callPackage ../games/legendary-gl { };
 
+  gogdl = python38Packages.callPackage ../games/heroic/gogdl.nix { };
```

Comment:
- `gogdl`, although created by the same people from Heroic, should be separate from the `heroic` package as it is possible to use it standalone. I think it would be better to keep it on its own directory instead.
-  Let's try to keep it on the alphabetical order